pkg/etw/eventdata.go

```diff
@@ -28,6 +29,13 @@ func (ed *eventData) writeString(data string) {
 	_ = ed.buffer.WriteByte(0)
 }
 
+// writeUnicodeString appends a string converted to UTF-16, including the null terminator, to the buffer.
+func (ed *eventData) writeUnicodeString(data string) {
+	unicode := utf16.Encode([]rune(data))
+	binary.Write(&ed.buffer, binary.LittleEndian, unicode)
+	ed.buffer.Write([]byte{0, 0})
```

Comment:
to suppress the linter on error checking (or you can add a `//nolint:errcheck` directive)
```suggestion
	_ = binary.Write(&ed.buffer, binary.LittleEndian, unicode)
	_, _ = ed.buffer.Write([]byte{0, 0})
```